outputformatter.go

```diff
@@ -67,30 +67,17 @@ func (this *outputFormatter) linksToText() {
 }
 
 func (this *outputFormatter) getOutputText() string {
```

Comment:
Can you describe what the intention is behind this change?   Is there a test case to reproduce issues with the existing code?    I see it adds a few more newlines to the text output, otherwise seems functionally equivalent.